### ISSUE TITLE
Question about the coarse labels of CIFAR-100N.

### ISSUE THREAD

**Issue by Ethanwl** [UCSC-REAL/cifar-10-100n#6]
I find that the number of the coarse labels of CIFAR-100N is only 12. 
My code is
```
import torch
noise_file = torch.load('/data/lwang/data/cifar-n/CIFAR-100_human.pt')
classes = list(set(noise_file["noisy_coarse_label"]))
class_num = len(classes)
print(classes)
print(class_num)
```
The output is
```
[1, 4, 7, 9, 10, 12, 14, 15, 16, 17, 18, 19]
12
```
Is there any wrong?

**Comment by weijiaheng** [comment]
Hi,

Thanks for pointing out this issue. We have fixed it and the file `CIFAR-100_human.pt` is not updated (with correct clean and noisy coarse labels).

The generation of coarse labels is based on the file [fine2coarse.py](https://github.com/UCSC-REAL/cifar-10-100n/blob/main/fine2coarse.py) which is provided in the repo as well.
